tests/src/test/scala/system/basic/WskBasicTests.scala

```diff
@@ -814,6 +877,37 @@ class WskBasicTests
         stderr should include regex (s"""Unable to get rule '$name': The requested resource does not exist. \\(code \\d+\\)""")
     }
 
+    it should "return a list of alphabetized rules" in withAssetCleaner(wskprops) {
```

Comment:
Thanks for the input! I may pick your brain about this later, as we had a different test involving Sequences that got a bit too complicated. Will definitely revisit!